History.md

```diff
@@ -25,6 +16,12 @@
   version of `npm` used by `meteor npm ...` commands.
   [PR #8835](https://github.com/meteor/meteor/pull/8835)
 
+* Windows now supports a 64-bit architecture using native 64-bit binaries.
+  In addition to hopefully being faster, this should allow Windows users to take
+  better advantage of the new V8 inspector API, which appears to only support
```

Comment:
Let's remove "hopefully" and change "appears to only support" to "only supports."

Reply:
Ok! Sticking to the firm facts!